test/signup.test.js

```diff
@@ -0,0 +1,168 @@
+const request = require('supertest'),
+  app = require('../app.js'),
+  Users = require('../app/models').user,
+  { factory } = require('factory-girl'),
+  validationErrorStatus = 401,
+  createdCorrectlyStatus = 200,
+  firstName = 'fn',
+  lastName = 'ln',
+  correctPassword = 'password',
+  correctEmail = 'email@wolox.com.ar';
+
+describe('Test invalid password', () => {
+  test('Create user with short password', () => {
+    expect.assertions(1);
```

Comment:
Remember that `expect.assertions(1)` is not necesary if you dont have a `catch()`. But you can leave it.